### PR TITLE
[keyboard] added swiftrax/cowfish

### PR DESCRIPTION
**Pull request by swiftrax**
## Description

- Added folder for my other keyboards
- Added keyboard called 'cowfish'

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/swiftrax/cowfish/readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+A TKL with F13 that uses Ai03's unified daughter board
+
+Keyboard Maintainer: Swiftrax 
+Hardware Supported: CowFish 
+Hardware Availability: https://github.com/swiftrax
+
+Make example for this keyboard (after setting up your build environment):
+
+    make cowfish:default
```
Comment (zvecr) [inline keyboards/handwired/swiftrax/cowfish/readme.md:12]:
```suggestion
# CowFish

A TKL with F13 that uses Ai03's unified daughter board

* Keyboard Maintainer: Swiftrax
* Hardware Supported: CowFish
* Hardware Availability: https://github.com/swiftrax

Make example for this keyboard (after setting up your build environment):

    make handwired/swiftrax/cowfish:default
```

File: keyboards/handwired/swiftrax/cowfish/rules.mk [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+CONSOLE_ENABLE = no	# Console for debug
+COMMAND_ENABLE = no    # Commands for debug and configuration
+NKRO_ENABLE = no		# USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+RGBLIGHT_ENABLE = no   # Enable keyboard underlight functionality
+BACKLIGHT_ENABLE = no  # Enable keyboard backlight functionality
+MIDI_ENABLE = no 		# MIDI controls
+AUDIO_ENABLE = no
+UNICODE_ENABLE = no 		# Unicode
+BLUETOOTH_ENABLE = no # Enable Bluetooth with the Adafruit EZ-Key HID
+
```
Comment (fauxpark) [inline keyboards/handwired/swiftrax/cowfish/rules.mk:29]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
```

File: keyboards/handwired/swiftrax/cowfish/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  _______, _______, _______,                            _______,                            _______, _______, _______,  _______,          _______, _______, _______),
+
+[2] = LAYOUT_all(
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______,   \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,   \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,  _______,  _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,                                       \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                                      _______,           \
+  _______, _______, _______,                            _______,                            _______, _______, _______,  _______,          _______, _______, _______)
```
Comment (fauxpark) [inline keyboards/handwired/swiftrax/cowfish/keymaps/via/keymap.c:27]:
```suggestion
  KC_ESC,  KC_F1,   KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,           KC_PSCR, KC_SLCK, KC_PAUS,
  KC_GRV,  KC_1,    KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSLS, KC_INS,  KC_HOME, KC_PGUP,
  KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_DEL,  KC_END,  KC_PGDN,
  KC_CAPS, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
  KC_LSFT, KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT,  KC_SLSH, KC_RSFT,                                      KC_UP,
  KC_LCTL, KC_LGUI, KC_LALT,                        KC_SPC,                          KC_RALT, KC_RGUI, KC_APP,  KC_RCTRL,          KC_LEFT, KC_DOWN, KC_RGHT),

[1] = LAYOUT_all(
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,  _______,  _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                                      _______,
  _______, _______, _______,                            _______,                            _______, _______, _______,  _______,          _______, _______, _______),

[2] = LAYOUT_all(
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,  _______,  _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                                      _______,
  _______, _______, _______,                            _______,                            _______, _______, _______,  _______,          _______, _______, _______)
```

File: keyboards/handwired/swiftrax/cowfish/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  KC_LCTL, KC_LGUI, KC_LALT,                        KC_SPC,                          KC_RALT, KC_RGUI, KC_MENU,  KC_RCTRL,        KC_LEFT, KC_DOWN, KC_RGHT),
+
+[_FN] = LAYOUT(
+  KC_ESC,  KC_F1,   KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,           KC_PSCR, KC_SLCK, KC_PAUS, \
+  KC_GRV,  KC_1,    KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSLS, KC_INS,  KC_HOME, KC_PGUP, \
+  KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_DEL,  KC_END,  KC_PGDN, \
+  KC_CAPS, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,                                      \
+  KC_LSFT, KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT,  KC_SLSH, KC_RSFT,                                     KC_UP,            \
+  KC_LCTL, KC_LGUI, KC_LALT,                        KC_SPC,                          KC_RALT, KC_RGUI, KC_MENU,  KC_RCTRL,        KC_LEFT, KC_DOWN, KC_RGHT)
```
Comment (fauxpark) [inline keyboards/handwired/swiftrax/cowfish/keymaps/default/keymap.c:28]:
```suggestion
  KC_ESC,  KC_F1,   KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,           KC_P1, KC_SLCK, KC_PAUS,
  KC_GRV,  KC_1,    KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSLS, KC_INS,  KC_HOME, KC_PGUP,
  KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_DEL,  KC_END,  KC_PGDN,
  KC_CAPS, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
  KC_LSFT, KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT,  KC_SLSH, KC_RSFT,                                     KC_UP,
  KC_LCTL, KC_LGUI, KC_LALT,                        KC_SPC,                          KC_RALT, KC_RGUI, KC_MENU,  KC_RCTRL,        KC_LEFT, KC_DOWN, KC_RGHT),

[_FN] = LAYOUT(
  KC_ESC,  KC_F1,   KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,  KC_F7,  KC_F8,  KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_F13,           KC_PSCR, KC_SLCK, KC_PAUS,
  KC_GRV,  KC_1,    KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSLS, KC_INS,  KC_HOME, KC_PGUP,
  KC_TAB,  KC_Q,    KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_DEL,  KC_END,  KC_PGDN,
  KC_CAPS, KC_A,    KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
  KC_LSFT, KC_Z,    KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT,  KC_SLSH, KC_RSFT,                                     KC_UP,
  KC_LCTL, KC_LGUI, KC_LALT,                        KC_SPC,                          KC_RALT, KC_RGUI, KC_MENU,  KC_RCTRL,        KC_LEFT, KC_DOWN, KC_RGHT)
```

File: keyboards/handwired/swiftrax/cowfish/rules.mk [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = qmk-dfu
```
Comment (Erovia) [inline keyboards/handwired/swiftrax/cowfish/rules.mk:12]: Is it running QMK DFU for sure? Or is it stock Atmel DFU?

Comment (swiftrax) [reply keyboards/handwired/swiftrax/cowfish/rules.mk:12]: stock amtel dfu. Should i change it to amtel?

File: keyboards/handwired/swiftrax/cowfish/rules.mk [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = amtel-dfu
```
Comment (fauxpark) [inline keyboards/handwired/swiftrax/cowfish/rules.mk:12]:
```suggestion
BOOTLOADER = atmel-dfu
```

Comment (Erovia) [reply keyboards/handwired/swiftrax/cowfish/rules.mk:12]: There is a typo here.

Comment (swiftrax) [reply keyboards/handwired/swiftrax/cowfish/rules.mk]: fixed it thanks

File: keyboards/handwired/swiftrax/cowfish/config.h [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x7161
+#define PRODUCT_ID      0x5239
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Swiftrax
+#define PRODUCT         CowFish
+#define DESCRIPTION     TKL with F13
+
+/* key matrix size */
+#define MATRIX_ROWS 8
```
Comment (zvecr) [inline keyboards/handwired/swiftrax/cowfish/config.h:31]:
```suggestion
#define MATRIX_ROWS 6
```

**Pull request merged**